gradebook.py

```diff
@@ -65,3 +65,29 @@ def view_grades(self, instructor, course_id):
             print_warning("No grades have been entered for this course yet.")
         
         input("Press enter to continue.")
+
+    def sort_courses(self, arrangement_type):
+        # Checks if dictionary is empty
+        if not self.grades:
+            print("Grades are empty. Please add a grade")
+            return
+
+        if arrangement_type != 'a' or arrangement_type != 'd':
+            print("Please type either (a/d)")
```

Comment:
All the print statements and the other print statements in this method will be `clear_screen` so it won't show on the terminal. Please run main.py manually and see if your option works.

Also, add a test in test_main.py to check the option 4 works as intended.